fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -149,14 +149,17 @@ def parse_line(line, hide_output)
     end
 
     def additional_upload_parameters
-      # Workaround because the traditional transporter broke on 1st March 2018
-      # More information https://github.com/fastlane/fastlane/issues/11958
-      # As there was no communication from Apple, we don't know if this is a temporary
-      # server outage, or something they changed without giving a heads-up
-      if ENV["DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS"].to_s.length == 0
-        ENV["DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS"] = "-t DAV,Signiant"
+      env_deliver_additional_params = ENV["DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS"]
```

Comment:
Can you add a comment above this with the new issue that this is fixing? 😊 I know that info will come in handy down the road

Reply:
Sure, will do within 2 hours

Reply:
Done, please take a look ;)

Reply:
This is perfect! Thank you ❤